src/main/java/com/redislabs/redisai/RedisAI.java

```diff
@@ -320,6 +320,34 @@ public boolean resetStat(String key) {
       throw new RedisAIException(ex);
     }
   }
+
+  /**
+   * AI.CONFIG <BACKENDSPATH <path>>
+   *
+   * @return
+   */
+  public boolean setBackendPath(String path) {
```

Comment:
@gkorland @dengliming should we set it to plural ( setBackendsPath )? given that the config property is BACKENDSPATH

Reply:
agree. fixed